documents/payment.md

```diff
@@ -766,6 +766,18 @@ please refer this [doc](https://razorpay.com/docs/payments/third-party-validatio
   "customer_name": "Gaurav Kumar"
 }
 ```
+-------------------------------------------------------------------------------------------------------
+### Fetch payment methods (Third party validation)
+
+```C#
+RazorpayClient client  = new RazorpayClient("key",""); // Use only razorpay key
```

Comment:
Have you tested if this works? I hope passing secret as null is not triggering the Oauth flow.

Reply:
Yes, I tested this and its working

![Screenshot 2024-05-06 132007](https://github.com/razorpay/razorpay-dot-net/assets/89454448/f202a889-b2fc-4b4c-a9db-099d2588a9c2)